syntax = "proto3";

enum MSGID
{
    MSG_BEGIN = 0;
    MSG_SERVER_REGISTER_REQ = 10;
    MSG_SERVER_REGISTER_RSP = 11;
    MSG_PLAYER_REGISTERTOGATE = 12;
    MSG_TEST_ID = 1005 ;
    MSG_PLAYER_MOVE_ID = 1006 ;
    MSG_PLAYER_ATTACK_ID = 1007 ;
    MSG_PLAYER_REGISTE_ID = 1008 ;
    MSG_PLAYER_LOGIN = 9999;
    
    MSG_ITEM_ADD = 10000;
    MSG_ITEM_REMOVE = 10001;
    MSG_PLAYER_FIND = 10002;
    MSG_RANKLIST_SHOW = 10003;

};
